config.yaml

```diff
@@ -1,14 +1,7 @@
 # Copyright 2021 Canonical Ltd.
 # See LICENSE file for licensing details.
-#
-# TEMPLATE-TODO: change this example to suit your needs.
-# If you don't need a config, you can remove the file entirely.
-# It ties in to the example _on_config_changed handler in src/charm.py
-#
-# Learn more about config at: https://juju.is/docs/sdk/config
-
 options:
-  thing:
-    default: 🎁
-    description: A thing used by the charm.
+  innodb_cluster_name:
```

Comment:
Since this seems to be something that can be set at runtime, it makes sense to put this in the config, and access it with `self.model.config['innodb_cluster_name']`. That gives operators a nice transparent way to set it.